src/packages/excalidraw/README_NEXT.md

```diff
@@ -590,7 +595,15 @@ Try out the [Demo](#Demo) to see it in action.
 
 ### detectScroll
 
-Indicates whether Excalidraw should listen for `scroll` event on the nearest scrollable container in the DOM tree and recompute the coordinates (e.g. to correctly handle the cursor) when the component's position changes. You can disable this when you either know this doesn't affect your app or you want to take care of it yourself (calling the [`refresh()`](#ref) method).
+Indicates whether Excalidraw should listen for `scroll` event on the nearest scrollable container in the DOM tree and recompute the coordinates (e.g. to correctly handle the cursor) when the component's position changes. You can disable this when you either know this doesn't affect your app or you want to take care of it yourself (calling the [`refresh()`](#refresh) method).
+
+### detectPosition
+
+Indicates whether the coordinates should be recomputed (eg to correctly handle the cursor) when the component's position changes. This will be disabled by default.
+
+Resizing and handling nearest scrollable parent ([detectScroll](#detectScroll)) is already handled so if there is any other case where position gets updated is where you will want to enable this prop (eg multiple scroll containers, or position updating due to addition/removal of elements in flex container).
+
+You might want to disable [detectScroll](#detectScroll) when you enable this prop.
```

Comment:
```suggestion

Position change detection is done by polling the DOM at a short interval which can impact performance. As such, this setting is disabled by default. In most apps, we believe that detecting position change on top of scroll and resize detection is also not needed.
```

Scroll detection is cheap in comparison, and is debounced. Therefore I'd not recommend this optimization. Instead, I'd explain the tradeoffs of detectScroll, hence the proposed edit